server/server.js

```diff
@@ -256,6 +262,35 @@ const fetchOrCreateUserToken = async () => {
   }
 };
 
+/**
+ * Simulates what Income precheck might look like if you were to run it with
+ * an employer that has a "HIGH" confidence level. This call only works
+ * in the sandbox environment.
+ */
+app.post("/appServer/simulate_precheck", async (req, res, next) => {
+  try {
+    if (process.env.PLAID_ENV !== "sandbox") {
+      res.status(500).json({
+        error: "This hard-coded example only works in the sandbox environment",
+      });
+      return;
+    }
+    const targetConfidence = req.body.confidence;
+    const employerName =
+      targetConfidence === "HIGH" ? "employer_good" : "Acme, Inc.";
```

Comment:
I'm a little confused as to what the hard coding of "Acme, Inc." does when you send it in the request body.  Just curious...can you explain?

Reply:
Basically, adding an unknown employer like this "resets" the pre-check back into an `UNKNOWN` state, which then lets you go through the full Income flow.